src/plugins/intel_cpu/src/node.cpp

```diff
@@ -562,18 +566,22 @@ void Node::updateDynamicParams() {
                     getTypeStr(),
                     " with name: ",
                     getName());
-    if (isExecutable()) {
-        if (needPrepareParams()) {
-            OPENVINO_ASSERT(inputShapesDefined(),
-                            "Can't prepare params for ",
-                            getTypeStr(),
-                            " node with name: ",
-                            getName(),
-                            " since the input shapes are not defined.");
-            DEBUG_LOG(" prepareParams() on #", getExecIndex(), " ", getTypeStr(), " ", algToString(getAlgorithm()),
-                      " ", getName(), " ", getOriginalLayers());
-            prepareParams();
+    try {
+        if (isExecutable()) {
+            if (needPrepareParams()) {
+                OPENVINO_ASSERT(inputShapesDefined(),
+                                "Can't prepare params for ",
+                                getTypeStr(),
+                                " node with name: ",
+                                getName(),
+                                " since the input shapes are not defined.");
```

Comment:
Nit. May be at least here it makes sense to remove the name and the type of the node as this information will be added anyway?